### PR TITLE
Fix README

### PR DESCRIPTION
**Pull request by derekchiang**
Since it was copied directly from Google docs there are some formatting errors.

### CODE REVIEW

**Comment by derekchiang** [comment]
Added a fix for 1) a typo (`in.aci` instead of `int.aci`) and 2) a wrong command (`add-image` instead of `add`)

File: README.md [hunk 1]
```diff
@@ -164,8 +164,8 @@ build mongodb from official images
 use apt-get to install nginx
 
 
-    acbuild -in=in.aci -out=int.aci add aptitude.aci 
-    acbuild -in=int.aci -out=out.aci exec apt-get -y install nginx
+    acbuild -in=in.aci -out=int.aci add-image aptitude.aci
+    acbuild -in=in.aci -out=out.aci exec apt-get -y install nginx
```
Comment (ecnahc515) [inline README.md]: the `-in` image should be `int.aci`, the output of the previous call, yes?

Comment (klizhentas) [inline README.md]: right, int.aci is just a typo

Comment (ecnahc515) [inline README.md]: Then the `-out=int.aci` in the first acibuild command should be `-out=in.aci`?

**Comment by jonboulle** [comment]
@derekchiang thanks for the patch! would you mind squashing your commits?

**Comment by derekchiang** [comment]
@jonboulle done

File: README.md (hunk 1 above)
Comment (jonboulle) [inline README.md]: actually, int was not a typo, it was intended to be an intermediate ACI.

**Pull request merged**